app/src/matchmaking/output/output_fastchess.hpp

```diff
@@ -20,17 +20,18 @@ class Fastchess : public IOutput {
 
     void printInterval(const SPRT& sprt, const Stats& stats, const std::string& first, const std::string& second,
                        const engines& engines, const std::string& book) override {
-        std::cout << "--------------------------------------------------\n" << std::flush;
-        printElo(stats, first, second, engines, book);
-        printSprt(sprt, stats);
-        std::cout << "--------------------------------------------------\n" << std::flush;
+        std::cout <<
+        "--------------------------------------------------\n" <<
+        printElo(stats, first, second, engines, book) <<
+        printSprt(sprt, stats) <<
+        "--------------------------------------------------\n" << std::flush;
```

Comment:
this still races.. you can't use multiple `<<` make it one string and then output @gahtan-syarif